src/test/java/com/mongodb/hibernate/internal/translate/mongoast/AbstractAstNodeTests.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.internal.translate.mongoast;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import org.bson.json.JsonWriter;
+
+public abstract class AbstractAstNodeTests {
+
+    protected void assertJsonRendered(AstNode node, String expectedJson) {
```

Comment:
Let's reorder the parameters to be consistent with the parameter order in JUnit5: expected (`expectedJson`) comes first, and the actual (`node`) comes second.